src/CONST.ts

```diff
@@ -2082,6 +2083,7 @@ const CONST = {
 
     ACCOUNT_ID: {
         ACCOUNTING: Number(Config?.EXPENSIFY_ACCOUNT_ID_ACCOUNTING ?? 9645353),
+        ACCOUNTS_PAYABLE: Number(Config?.EXPENSIFY_ACCOUNT_ID_ACCOUNTS_PAYABLE ?? 10903701),
```

Comment:
Its the correct accountId
<img width="1182" alt="image" src="https://github.com/user-attachments/assets/6a7fb923-f86b-41f4-8b56-854ea537b46e">